src/Gnugat/Medio/Model/Method.php

```diff
@@ -188,6 +198,11 @@ public function removeStatic()
      */
     public function setBody($body)
     {
+        
+        if ($this->isAbstract()) {
+            throw new \Gnugat\Medio\Exception\DomainException("You can`t set body to abstract method: " . $this->getName());
```

Comment:
You could add a use statement to avoid having to write the exception's fully qualified classname (we'd be able to see the whole message without scrolling, and by looking at the top of the file we'd be able to tell the dependencies).

Coding standard: no spaces around the concatenation operator